webapp/src/components/channel_header_button/index.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {isPluginRhsOpen} from 'selectors';
+
+import ChannelHeaderButton from './channel_header_buttons.tsx';
```

Comment:
You can remove the extension here, and I think it should be named singular `channel_header_button`.

```suggestion
import ChannelHeaderButton from './channel_header_button';
```